oktoberfest/ce_calibration.py

```diff
@@ -47,13 +48,25 @@ def __init__(
         :param config_path: path to configuration file
         :param mzml_reader_package: mzml reader (pymzml or pyteomics)
         """
+        if isinstance(search_path, str):
+            search_path = Path(search_path)
```

Comment:
since this is forwarded to the superclass, which is handling conversion and also accepts Union[str, Path], you should remove this here and only handle it in the superclass.